packages/date/stories/date.stories.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-console */
 import React from 'react';
-import { Meta, Story } from '@storybook/react';
+import { StoryObj } from '@storybook/react';
 import { Button } from 'reactstrap';
 import { avDate, dateRange } from '@availity/yup';
 import * as yup from 'yup';
```

Comment:
Can we also try importing the date css file, looks like storybook isn't detecting it anymore